gateways/Gateways.md

```diff
@@ -13,18 +13,24 @@ All fjåge Gateway implementations should implement the following classes and me
 - Must return a `null` if the connection to master container fails on the first attempt.
 - May support auto-reconnect, where, once a connection with the master container is established if the connection fails, the Gateway tries to reconnect automatically.
 
-### `getAgentID()` :: Void -> AgentID
+### `authenticate` :: String credentials -> Boolean
+- Tries to authenticate the Gateway connection with the master container.
+- Must send the credential string using command `{"action":"auth","creds":"boo"}`
```

Comment:
Do we need a comment on what `creds` may contain?

Reply:
That would depend on the implementation of the Firewall that is being used.

Reply:
Yes, but just a comment on this to state so?